prime-dotnet-webapi/Controllers/OrganizationsController.cs

```diff
@@ -110,6 +123,116 @@ public async Task<ActionResult<Organization>> CreateOrganization(CreateOrganizat
             );
         }
 
+        // POST: api/Organizations/claim
+        /// <summary>
+        /// Claim an existing Organization.
+        /// </summary>
+        [HttpPost("claim", Name = nameof(ClaimOrganization))]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(typeof(ApiResultResponse<int>), StatusCodes.Status200OK)]
+
+        public async Task<ActionResult<Organization>> ClaimOrganization(ClaimOrganizationViewModel claimOrganization)
+        {
+            if (!await _partyService.PartyExistsAsync(claimOrganization.PartyId, PartyType.SigningAuthority))
+            {
+                return BadRequest("Could not claim an organization, the passed in SigningAuthority does not exist.");
+            }
+
+            var organization = await _organizationService.ClaimOrganizationAsync(claimOrganization.PartyId, claimOrganization.PEC, claimOrganization.ClaimDetail);
+
+            return Ok(organization);
+        }
+
+        // GET: api/Organizations/claim
+        /// <summary>
+        /// Gets organization claim count by a given search criteria.
+        /// </summary>
+        [HttpGet("claim", Name = nameof(GetOrganizationClaimCount))]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(ApiResultResponse<OrganizationClaim>), StatusCodes.Status200OK)]
+        public async Task<ActionResult<int>> GetOrganizationClaimCount([FromQuery] OrganizationClaimSearchOptions search)
+        {
+            // check if the organization exists with the given PEC
+            if (!string.IsNullOrEmpty(search.Pec))
+            {
+                var organization = await _organizationService.GetOrganizationByPecAsync(search.Pec);
+                if (organization == null)
+                {
+                    return BadRequest("Organization does not exist with the passed in site PEC.");
+                }
+            }
+            var result = await _organizationService.GetOrganizationClaimCountAsync(search);
+
+            return Ok(result);
+        }
+
+        // GET: api/Organizations/claim/5
+        /// <summary>
+        /// Find OrganizationClaim by Organization ID.
+        /// </summary>
+        [HttpGet("claim/{organizationId}", Name = nameof(GetOrganizationClaimByOrgId))]
```

Comment:
Based on my other comment, it makes more sense to think of claims as (noun) collections on Organizations, so this URL should be /organizations/{organizationId}/claims